apps/editing-toolkit/editing-toolkit-plugin/tutorials/class-wpcom-tutorials.php

```diff
@@ -0,0 +1,246 @@
+<?php
+/**
+ * Tutorials (We would have called them Guided Tours but it's been done).
+ *
+ * Each Tutorial contains a flat list of Tasks. Each Task may contain several Steps in
+ * a `WpcomTourKit` component on the frontend, but we are only concerned with the top
+ * two levels of Tutorial and its contained Tasks, tracking Task completion status
+ * on the backend. This API will be used to generate
+ *
+ * @package A8C\FSE
+ */
+
+/**
+ * WPCom_Tutorials class
+ */
+class WPCom_Tutorials {
+
+	/**
+	 * Storage for our singleton instance.
+	 *
+	 * @var null|WPCom_Tutorials
+	 */
+	private static $instance = null;
+
+	/**
+	 * Internal storage for registered Tutorials.
+	 *
+	 * @var array
+	 */
+	private $registry = array();
+
+	/**
+	 * Gets aour singleton instance, initiating it the first time.
```

Comment:
`Jeremy the human Linter` 🤖 is wondering if we could update this comment to "Gets our..." instead of "Gets aour"

Reply:
aour! (sounds made by cats up to unmentionable things)

Reply:
[aour!](https://github.com/Automattic/wp-calypso/pull/64510/commits/764e9b1fbc9dc6335c6a3370126ff1bc8d72c635)